sway/input/input-manager.c

```diff
@@ -542,6 +542,21 @@ void input_manager_apply_input_config(struct input_config *input_config) {
 	retranslate_keysyms(input_config);
 }
 
+void input_manager_configure_input(struct sway_input_device *input_device) {
```

Comment:
Can you replace the matching lines in `input_manager_apply_input_config` right above this with a call to this new function?

Reply:
Done :+1: